### PR TITLE
Function 'omited'

### PR DESCRIPTION
**Pull request by JeremyCoutellec**

### CODE REVIEW

File: src/tools.js [hunk 1]
```diff
@@ -60,6 +60,22 @@ export function isValidDate(date) {
   return !isNaN(date.getTime())
 }
 
+export function omited(propNamesOrMap) {
```
Comment (davidbonnet) [inline src/tools.js]:
```suggestion
export function omitted(propNames) {
```

File: src/tools.js [hunk 2]
```diff
@@ -60,6 +60,22 @@ export function isValidDate(date) {
...
+  /*
+  Returns a function that returns all props without a subset of the provided object.
+
+  Examples:
+
+    // Only omit the `value` prop
+    mapProps(omited(['value']))
+    
+  */
+  if (isArray(propNamesOrMap)) {
```
Comment (davidbonnet) [inline src/tools.js]: It is always an array.

File: src/tools.js [hunk 3]
```diff
@@ -60,6 +60,22 @@ export function isValidDate(date) {
...
+    return (props) => omit(props, propNamesOrMap)
+  }
+  return props
```
Comment (davidbonnet) [inline src/tools.js]: Can't be, `props` is not defined.

File: src/tools.js [hunk 4]
```diff
@@ -60,6 +60,22 @@ export function isValidDate(date) {
   return !isNaN(date.getTime())
 }
 
+export function omited(propNamesOrMap) {
+  /*
+  Returns a function that returns all props without a subset of the provided object.
```
Comment (davidbonnet) [inline src/tools.js]:
```suggestion
  Returns a function that returns all props without the ones whose name is in `propNames`.
```

File: src/tools.js [hunk 5]
```diff
@@ -60,6 +60,22 @@ export function isValidDate(date) {
...
+
+  Examples:
+
+    // Only omit the `value` prop
+    mapProps(omited(['value']))
```
Comment (davidbonnet) [inline src/tools.js]:
```suggestion
    mapProps(omitted(['value']))
```

File: src/tools.js [hunk 6]
```diff
@@ -60,6 +60,22 @@ export function isValidDate(date) {
   return !isNaN(date.getTime())
 }
 
+export function omited(propNamesOrMap) {
+  /*
+  Returns a function that returns all props without a subset of the provided object.
+
+  Examples:
```
Comment (davidbonnet) [inline src/tools.js]:
```suggestion
  Example:
```

File: src/tools.js [hunk 7]
```diff
@@ -1,5 +1,5 @@
 import { createElement, isValidElement } from 'react'
-import { every, memoize, pick, mapValues, get } from 'lodash'
+import { every, memoize, pick, mapValues, get,omit } from 'lodash'
```
Comment (davidbonnet) [inline src/tools.js]: `npm run prettier`

File: src/tools.js [hunk 8]
```diff
@@ -60,6 +60,19 @@ export function isValidDate(date) {
   return !isNaN(date.getTime())
 }
 
+export function omitted(propNames) {
+  /*
+  Returns a function that returns all props without the ones whose name is in `propNames.
```
Comment (davidbonnet) [inline src/tools.js]: Doc should also be added to the `README.md`.

Comment (JeremyCoutellec) [reply src/tools.js]: on another pr

**Pull request closed without merging**